### PR TITLE
Fixes infinite loop when rendering chart

### PR DESCRIPTION
**Pull request by aroc**
## What does this PR do? How does it affect users?

It fixes a bug where Explorer would get into an infinite loop when rendering the chart. This was because we were using state to keep track of the lastDataTimestamp, which caused re-renders. Instead, we're just using a component property.

## How should this be tested?

Try loading different saved queries one after the other and ensure they do not cause an infinite loop inside of `showVisualization` inside of the `KeenViz` component.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:

- [ ] Does the code compile without warnings (check shell, console)?
- [x] Do all tests pass?
- [ ] Does the UI, pixel by pixel, look exactly as expected (check various screen sizes, including mobile)?
- [x] If the feature makes requests from the browser, inspect them in the Web Inspector. Do they look as expected (parameters, headers, etc)?
- [ ] If the feature sends data to Keen, is the data visible in the project if you run an extraction (include link to collection/query)?
- [ ] If the feature saves data to a database, can you confirm the data is indeed created in the database?

### CODE REVIEW

File: client/js/app/components/explorer/visualization/keen_viz.js [hunk 1]
```diff
@@ -7,11 +7,14 @@ var React = require('react');
...
 var KeenViz = React.createClass({
 
+  lastDataTimestamp: null,
+
 	// ***********************
 	// Convenience functions
 	// ***********************
 
 	showVisualization: function() {
+    console.log(new Date(), 'INSIDE showVisualization!')
```
Comment (joannecheng) [inline client/js/app/components/explorer/visualization/keen_viz.js]: Accidental `console.log` statement?

**Comment by joannecheng** [comment]
I don't see an infinite loop anymore when the chart renders, good job! Just need to remove the extra console.log statments.

File: client/js/app/components/explorer/visualization/keen_viz.js (hunk 1 above)
Comment (aroc) [inline client/js/app/components/explorer/visualization/keen_viz.js]: Nooooooo!!!!!
